### PR TITLE
Update cmd_bar_hotkeys.lua

### PR DESCRIPTION
**Pull request by A-childs-encyclopedia**
cc @badosu

### CODE REVIEW

File: luaui/Widgets/cmd_bar_hotkeys.lua [hunk 1]
```diff
@@ -614,7 +614,7 @@ local function makeBindsTable(keyLayout)
         table.insert(_binds, { 'Ctrl+'..i       , "group", "set "..i        })
         table.insert(_binds, { 'Shift+'..i      , "group", "selectadd "..i })
         table.insert(_binds, { 'Ctrl+Shift+'..i , "group", "add "..i        })
-        table.insert(_binds, { 'Alt+'..i        , "group", "selecttoggle "..i     })
+        table.insert(_binds, { 'Alt+Shift+'..i        , "group", "selecttoggle "..i     })
```
Comment (badosu) [inline luaui/Widgets/cmd_bar_hotkeys.lua:617]: Can you replace `Ctrl+Alt` with `Alt+Shift` for the windows users?

Personally I prefer `Ctrl+Shift` but we can't have default keybindings specific for minority operating systems.

Comment (A-childs-encyclopedia) [reply luaui/Widgets/cmd_bar_hotkeys.lua:617]: But Ctrl+Alt isn't there

Comment (A-childs-encyclopedia) [reply luaui/Widgets/cmd_bar_hotkeys.lua]: Done

**Comment by Beherith** [comment]
Badosu ty for noticing alt-shift, do ping if mergeable

**Comment by badosu** [comment]
Merge with squash!

**Pull request merged**